src/main/java/Duke.java

```diff
@@ -1,10 +1,346 @@
+
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import duke.task.Todo;
+import duke.task.Task;
+import duke.task.TestTask;
+import duke.task.Event;
+import duke.task.Deadline;
+import duke.exception.DukeException;
+
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class Duke {
+    static int listLength;
+
+    static Task task;
+
+
+    static String userInput;
+    static String command;
+    static String description = "No description is needed for this command";
+    public static final String SEPARATOR = "-----------------------------------";
+    private static ArrayList<Task> tasks;
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
+
         System.out.println("Hello from\n" + logo);
+
+
+
+
+
```

Comment:
In general, there are too many blocks of empty lines across your file. You could remove those to improve the readability of your code.  

As a guideline, leave at most 1 empty line to separate the functional blocks.